styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@media only screen and (max-width: 600px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    width: 100vw;
+    height: 3rem;
+    padding: 1rem;
+  }
+
+  .navbar .shortcuts {
+    display: none;
+  }
+
+  .options {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+
+  .social i {
+    font-size: 2rem;
+    color: rgb(59, 59, 59);
+  }
+
+  .options i {
+    font-size: 1.5rem;
+  }
+
+  .social i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  .options i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  #homepage {
+    display: flex;
+    max-width: 100vw;
+    min-height: 100vh;
+    background-image: url(header-illsutration-mobile@2x.png);
+    background-size: 100vw;
+    border-bottom-left-radius: 2rem;
+    flex-direction: column;
+    padding: 2rem;
+    justify-content: space-around;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/61361037/212145790-638a86e6-5320-4176-858a-d838de6a21bf.png)

The background image is repeating here. You can add `background-repeat: no-repeat;` to stop the repetition of the background.